### DISCUSSION TITLE
针对工单查询权限的需求和想法

### DISCUSSION THREAD

**Discussion by jinhaozcp** [hhyo/Archery#1639] [Ideas]
目前工单查询审批流中，经常出现申请的请求范围过大，如一次性针对某个数据库申请一年，或者针对某个表申请单词查询跳出几千条，说了很多次，都不改，不审批吧，又说我们阻碍测试。
为了安全起见，能否由审批人更改用户的查询审批要求，比如他申请一年，我们在审批的时候给他打个折，只给他1个月，或者他申请500条，我改为500条，然后放通审批。
这样即不阻碍他的研发测试流程，又能有效控制范围。望采纳。

**Comment by LeoQuote** [comment]
过了之后是可以改的, 你觉得这个符合需求吗?
